extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -18,15 +18,16 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	window.googletag = window.googletag || {};
 	window.googletag.cmd = window.googletag.cmd || [];
 
+	function isSlotRegistered(slot, id) {
+		return registeredCallbacks[id] && slot && slot === googleSlots.getSlot(id);
```

Comment:
ok, I see it is moved from `&& event.slot &&` but still the question is valid, do we need it?